lib/gobstones/spec/example.rb

```diff
@@ -0,0 +1,74 @@
+module Gobstones::Spec
+  class Example
+    include Gobstones::WithTempfile
+
+    attr_reader :gobstones_path
+
+    def initialize(gobstones_path)
+      @gobstones_path = gobstones_path
+    end
+
+    def start!(source_file, initial_board, final_board)
```

Comment:
I still don't get where to use the bang (!) and where not... can you shed some light on this?